src/main/java/com/minecraftabnormals/upgrade_aquatic/core/events/EntityEvents.java

```diff
@@ -181,6 +184,16 @@ public static void onPlayerMount(EntityMountEvent event) {
 		}
 	}
 	
+    @SubscribeEvent
+    public static void onDrownedPoseChange(EntityEvent.EyeHeight event) {
```

Comment:
Didn't notice this before, but this entire method isn't aligned with the other methods in this class.